docs/resources/aws_ec2_transit_gateway_route_table_association.md

```diff
@@ -0,0 +1,81 @@
+---
+title: About the aws_ec2_transit_gateway_route_table_association Resource
+platform: aws
+---
+
+# aws\_ec2\_transit\_gateway\_route\_table\_association
+
+Use the `aws_ec2_transit_gateway_route_table_association` InSpec audit resource to test properties of a single AWS transit gateway route table association.
+
+A transit gateway route table association associates the specified attachment with the specified transit gateway route table. You can associate only one route table with an attachment.
+
+## Syntax
+
+Ensure that a transit gateway route table ID exists.
+
+    describe aws_ec2_transit_gateway_route_table_association(transit_gateway_route_table_id: 'TRANSIT_GATEWAY_ROUTE_TABLE_ID') do
+      it { should exist }
+    end
+
+## Parameters
+
+`transit_gateway_route_table_id` _(required)_
+
+For additional information, see the [AWS documentation on transit gateway route table association](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-transitgatewayroutetableassociation.html).
```

Comment:
<!--StartFragment-->For additional information, check out the [AWS documentation on transit gateway route table association](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-transitgatewayroutetableassociation.html).<br class="Apple-interchange-newline"><!--EndFragment-->